rbac/default-role-manager/role_manager.go

```diff
@@ -24,13 +25,15 @@ import (
 	"github.com/casbin/casbin/v2/util"
 )
 
-const defaultDomain string = "casbin::default"
+const defaultDomain string = "casbin_default"
```

Comment:
Why `defaultDomain` is not empty string?

I think a global user alice should be ``::alice``